src/Components/SourcesSelect/index.js

```diff
@@ -65,20 +65,27 @@ const SourcesSelect = ({ setValidation, image }) => {
   }
 
   return (
-    <Select
-      ouiaId="select_account"
-      isOpen={isOpen}
-      onToggle={(openState) => setIsOpen(openState)}
-      selections={selected}
-      maxHeight="180px"
-      // TODO decide if to disable the select
-      // isDisabled={sources?.length === 1}
-      onSelect={onSelect}
-      placeholderText="Select account"
-      aria-label="Select account"
-    >
-      {sources && selectItemsMapper()}
-    </Select>
+    <>
+      <Select
+        ouiaId="select_account"
+        isOpen={isOpen}
+        onToggle={(openState) => setIsOpen(openState)}
+        selections={selected}
+        maxHeight="180px"
+        // TODO decide if to disable the select
+        // isDisabled={sources?.length === 1}
+        onSelect={onSelect}
+        placeholderText="Select account"
+        aria-label="Select account"
+      >
+        {sources && selectItemsMapper()}
+      </Select>
+      {selected != undefined && sources.find((source) => source.id === selected.id && source.status != 'available') && (
+        <HelperText id="account-error-inline">
+          <HelperTextItem variant="warning">source is unavailable</HelperTextItem>
```

Comment:
Would we want to include a link to sources, or any other resource to how to fix that?

Reply:
Sure, what text would we want to write here? 
source is unavailable, for more information click [here](https://console.redhat.com/settings/sources)?

Reply:
yeah, exactly! I'd maybe link to the source details (specific source) directly, but apart from that :+1: